modules/tinymce/src/themes/silver/test/ts/browser/throbber/ThrobberTest.ts

```diff
@@ -1,5 +1,5 @@
 import { ApproxStructure, Assertions, UiFinder, Waiter } from '@ephox/agar';
-import { describe, it } from '@ephox/bedrock-client';
+import { assert, describe, it } from '@ephox/bedrock-client';
```

Comment:
Don't use the legacy `assert` module, use chai instead please.

- [ ] Use chai assertions